corehq/apps/domain/views.py

```diff
@@ -657,7 +659,12 @@ def plan(self):
                     next_subscription.update({
                         'exists': True,
                         'date_start': subscription.next_subscription.date_start.strftime("%d %B %Y"),
+                        'name': subscription.next_subscription.plan_version.plan.name,
+                        'price': self.get_product_summary(subscription.next_subscription.plan_version,
+                                                          self.account,
+                                                          subscription)[0]['monthly_fee'],
```

Comment:
recommend flagging it in an error email with actionable text. example:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/models.py#L1106